### ISSUE TITLE
Final optimisation 2D-CNN

### ISSUE THREAD

**Issue by leo92ra** [tamap94/ds-capstone-alzheimers-#110]
### Dennis' notes:
**changes in pre-processing**
- [x] try using few multiple slices (treat same as channels)

**optimise the model**
- [x] cyclic learning rate

**changes to the training-set**
- [x] undersampling to deal with data imbalance (remove only young to fight both issues)

### Velentin's notes:
**try next**
- [x] modify get_slice functions to allow loading of thin 3D images (3 layers, but connected like in an RGB image)
- [x] can eigenbrains increase training time? do CNNs work with eigenbrains?

### Meeting with coaches:
**sort by priority and try**
- [x] add a dropout layer after each layer to improve the generalisation of the CNN
- [ ] denoising: use a denoising network before the CNN
- [x] set upper layer trainable in VGG16 (2-3 layers)
- [x] add more convolutional layers
- [x] add more pooling layers
- [x] check TensorFlow blog for segmentation/normalisation of medical images
https://blog.tensorflow.org/2018/07/an-introduction-to-biomedical-image-analysis-tensorflow-dltk.html
- [x] use more epochs
